packages/front-end/components/Features/AttributeModal.tsx

```diff
@@ -65,52 +68,51 @@ export default function AttributeModal({ close, attribute }: Props) {
           value.hashAttribute = false;
         }
 
-        const attributeSchema = [...schema];
-
-        // Editing
-        if (attribute) {
-          const i = schema.findIndex((s) => s.property === attribute);
-          if (i >= 0) {
-            attributeSchema[i] = value;
-          } else {
-            attributeSchema.push(value);
-          }
-        }
-        // Creating
-        else {
-          attributeSchema.push(value);
-        }
-
         // Make sure this attribute name doesn't conflict with any existing attributes
-        if (
-          attributeSchema.filter((s) => s.property === value.property).length >
-          1
-        ) {
+        if (!attribute && schema.some((s) => s.property === value.property)) {
           throw new Error(
             "That attribute name is already being used. Please choose another one."
           );
         }
 
-        const settings: Pick<OrganizationSettings, "attributeSchema"> = {
-          attributeSchema,
-        };
-        await apiCall(`/organization`, {
-          method: "PUT",
+        const url = attribute ? `/attribute/${attribute}` : `/attribute`;
```

Comment:
We don't have any validation on attribute name today, so someone could define an attribute with a slash in it or other non-URL safe characters.  It's not safe to pass this in the URL